app/src/main/java/com/x8bit/bitwarden/data/autofill/accessibility/processor/BitwardenAccessibilityProcessorImpl.kt

```diff
@@ -28,10 +29,16 @@ class BitwardenAccessibilityProcessorImpl(
         val rootNode = rootAccessibilityNodeInfo ?: return
         // Ignore the event when the phone is inactive
         if (!powerManager.isInteractive) return
-        // We skip if the package is not supported
-        if (rootNode.shouldSkipPackage) return
-        // We skip any package that is a launcher
-        if (launcherPackageNameManager.launcherPackages.any { it == rootNode.packageName }) return
+        // We skip if the system package
+        if (rootNode.isSystemPackage) return
```

Comment:
I had to split this out to make sure we do not clear the action when on the system package since that is where the tile lives.